CONTRIBUTING.md

```diff
@@ -24,6 +24,23 @@ Check out our [README.md] for more details.
 
     * The [InnerSource Commons Slack] [#ispo-working-group] community will post the next meeting.
 
+## Our Artifacts
+
+In addition to our [Kanban board] cards, we produce the following artifacts.
+
+```mermaid
+graph LR
+    WG[ISPO Working Group] --> GD[GitHub Discussions]
+    WG --> IP[InnerSource Patterns]
+    WG --> MI[Managing InnerSource Projects]
+    WG --> DR[ISPO Working Group GDrive]
+```
+
+* [GitHub Discussions] — A place to collect ideas for ❓.
+* [InnerSource Patterns] — Join in creating patterns that enhance the overall standards for InnerSource Practitioners.
+* [Managing InnerSource Projects] — Articles, KPIs, and other resources for managing InnerSource projects.
+* [ISPO Working Group GDrive] — Slides, spreadsheets, and other artifacts not easily represented elsewhere.
+
 ## New Content
```

Comment:
I looked at this section and here's my take.

We can make the upper section describe the "what", and this section the "how".

Does that sound like a good approach to simplify guidance?

If so, I'll refactor both sections and separate what artifacts we build and how we build them.

I'll can keep the section names the same or we can try **Contributing New Content** to make it easier to follow for someone coming direction to the section.

Reply:
Upon further inspection, it looks like the **3 Ways to Contribute** section also has overlaps.

I can take a stab at consolidating all three.

Any objections?